### PR TITLE
[TM] TG Update: Combat mode, drone exploration, and more!

### PR DESCRIPTION
**Pull request by SgtHunk**
## About The Pull Request

do not even merge this please not until i'm done

TO DO:

- [x] Cry
- [x] Cry more
- [x] Get a headache
- [x] Fix HUDs for fulp jobs
- [x] Tweak brig doc access to match the PR
- [x] Fix beefman limb-mashing (Combat mode consequence)
- [x] Fix beefman trauma (IT BROKEEEEEEEEEEE)
- [x] Pray Heliostation areas stop runtiming
- [x] Add all of the Robo-tourist stuff to Helios

## Heliostation

Major changes:
pharmacy
service
hfr
drone
experi-sci

Other changes were:
- Spare ID Safe in Bridge.
- Fixed a few atmos differences (mainly in engineering).
- Bluespace Gas Sender (in atmos) and Bluespace Gas Vendors (around the station).
- Hangover spawns.
- Brig now has firelocks.
- Interrogation room is now its own thing.
- All ID Consoles replaced with modular consoles.
- N-Spectors and Crowbar in Brig.
- Holodeck now has lights.

## Why It's Good For The Game

im going to sleep

## Changelog
:cl:
add: Updates the repository to the latest Tgstation commit update, keeping all fulp-only features. Yes, this includes Combat mode, Robotourists, Cargo Drones, and more.
/:cl:

### CODE REVIEW

**Comment by GuillaumePrata** [comment]
Thanks Pucci I will test merge this right now then

File: README.md [hunk 1]
```diff
@@ -1,17 +1,21 @@
-## Fulpstation codebase
```
Comment (JohnFulpWillard) [inline README.md:1]: Gotta revert this

File: fulp_modules/jobs/brigdoc/code/brigdoc.dm [hunk 2]
```diff
@@ -51,9 +50,20 @@
...
 	gloves = /obj/item/clothing/gloves/color/plasmaman/white
+
+/datum/id_trim/job/brig_physician
+	assignment = "Brig Physician"
+	trim_icon = 'fulp_modules/jobs/cards.dmi'
+	trim_state = "trim_brigphysician"
+	access = list(ACCESS_SECURITY, ACCESS_SEC_DOORS, ACCESS_BRIG, ACCESS_MEDICAL, ACCESS_MORGUE, ACCESS_SURGERY, ACCESS_PHARMACY, ACCESS_CHEMISTRY, ACCESS_MECH_MEDICAL, ACCESS_MINERAL_STOREROOM, ACCESS_WEAPONS)
+	minimal_access = list(ACCESS_SEC_DOORS, ACCESS_BRIG, ACCESS_MEDICAL, ACCESS_MORGUE, ACCESS_SURGERY, ACCESS_MECH_MEDICAL, ACCESS_MINERAL_STOREROOM, ACCESS_WEAPONS)
+	config_job = "brig_physician"
+	template_access = list(ACCESS_CAPTAIN, ACCESS_CMO, ACCESS_HOS, ACCESS_CHANGE_IDS)
```
Comment (JohnFulpWillard) [inline fulp_modules/jobs/brigdoc/code/brigdoc.dm:69]: Does this mean CMO can give Brig access, and HoS can give Medbay access?

**Comment by SgtHunk** [comment]
CI please save me

**Comment by StoneMason** [comment]
7f0dddb44ddea833d0962148d70a938b

**Comment by SgtHunk** [comment]
what did he mean by this

**Comment by JohnFulpWillard** [comment]
> 
> 
> 
> 
> what did he mean by this

probably because he also has a PR bringing over a few PRs from TG

**Comment by urococt** [comment]
> what did he mean by this

I'm salty about a lot of the changes TG has made. Please ignore me.

**Comment by AffectedArc07** [comment]
This PR crashed mapdiffbot for 35 minutes.

You have been blacklisted from the service.

**Comment by JohnFulpWillard** [comment]
> 
> 
> This PR crashed mapdiffbot for 35 minutes.
> 
> You have been blacklisted from the service.

God damn, is Heliostation that powerful?

File: code/__DEFINES/logging.dm [hunk 3]
```diff
@@ -1,54 +1,54 @@
 //Investigate logging defines
```
Comment (JohnFulpWillard) [inline code/__DEFINES/logging.dm:1]: now that i think about it, I dont think mentor logs are actually visible. maybe its worth adding

File: code/datums/datacore.dm [hunk 4]
```diff
@@ -152,7 +152,7 @@
 	)
 	var/list/departments = list(
 		"Command" = GLOB.command_positions,
-		"Security" = GLOB.security_positions,
+		"Security" = GLOB.security_positions + GLOB.security_sub_positions,
```
Comment (JohnFulpWillard) [inline code/datums/datacore.dm:155]: pain

**Comment by JohnFulpWillard** [comment]
i think you have conflicts... you should fix that...

**Comment by Qustinnus** [comment]
if the update includes it, don't forget to map in experi-sci.

**Comment by SgtHunk** [comment]
All of the mapping stuff should be included unless I missed something
Fermichem heaters, Experi-sci, the NanoDrug (which was forgotten), the spare ID safe, etc

**Comment by Qustinnus** [comment]
and hangover spawn landmarks, they are the ones that look like bottles? they are quite new.

**Comment by SgtHunk** [comment]
Also already mapped

**Comment by Qustinnus** [comment]
based, good luck with this pr.

**Comment by CyoTheProtogen** [comment]
insanity
he's gone insane

**Comment by remove32** [comment]
fix the changelog you brat

**Pull request merged**